components/log-viewer-webui/server/src/plugins/MongoSocketIoServer/index.ts

```
@@ -0,0 +1,392 @@
+
+        const lastSubscriber = collection.unsubcribe(queryId, socket.id);
+        this.#fastify.log.info(`Socket ${socket.id} unsubscribed from query ${queryId}`);
+
+        if (lastSubscriber) {
+            this.#fastify.log.info(`QueryID:${queryId} deleted from query map.`);
+            this.#queryIdtoQueryHashMap.delete(queryId);
+        }
+
+        this.#fastify.log.debug(
+            "Query ID to query hash map:" +
+            ` ${JSON.stringify(Array.from(this.#queryIdtoQueryHashMap.entries()))}`
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)